### ISSUE TITLE
Missing Shapes

### ISSUE THREAD

**Issue by sterlingbaldwin** [NintendoExaminer/Lets-Make-A-Voxel-Game#1]
Im trying to get this working, and it seems like there needs to be at least one basic cube shape. In the MeshBuilder ```ShapeManager.shapes[data.shapeID].GetFaces(visibleFaces, faceData);``` pulls in the face data. Unfortunately there are no shapes in the shapes directory. Without knowing the format that the shapes are expected to be in its hard to make this work. Would it be possible to get a single cube shape so we can make our own in the given format?

**Comment by Jump-Suit** [comment]
I would love for their to be a template for this as well as progress being picked up on this.

**Comment by sterlingbaldwin** [comment]
IDK if the author is planning on continuing work in this repo. I watched the first couple videos in the Lets Make A Voxel Game series and thought it was really interesting. Ive made a repo with working versions of the state after each video for the first couple episodes, basically enough to bootstrap getting something working. https://github.com/sterlingbaldwin/Lets-Make-A-Voxel-Game

**Comment by Jump-Suit** [comment]
Yes I am interested in this project as well. I have watched pretty much the whole serious and if you didn't know he overhauls that completely for managers and the like, and btw you CAN get his second .zip: caves and better terrain gen by using the Lightweight Render Pipeline, if you can help me sort out the Shapes for this I can get a functional build working. He has it setup as a RTS camera but i am going to revert that back to the 1st person controller as it should have been. and move from there.
